src/ensembl/src/content/app/browser/track-panel/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -50,7 +50,8 @@ const getTranscriptTrackId = (num: number) => `track:transcript-feat-${num}`;
 
 const TrackPanelGene = (props: TrackPanelGeneProps) => {
   const { genomeId, geneId, ensObjectId } = props;
-  const [isCollapsed, setIsCollapsed] = useState(true);
+  const startWithCollapsed = !isEnvironment([Environment.PRODUCTION]); // TODO: remove after multiple transcripts are available
```

Comment:
Since we are calling `isEnvironment([Environment.PRODUCTION])` in 3 different places, can we define  `const isProduction = isEnvironment([Environment.PRODUCTION])` separately and use it here and in the lines below?

Reply:
All these isEnvironment lines are going to be removed in foreseeable future. If we define `const isProduction` as `isEnvironment([Environment.PRODUCTION])`, it will be one more line to remove :-) It won't really help us much.